packages/godwoken/src/client.ts

```diff
@@ -91,6 +92,14 @@ export class GodwokenClient {
     return +nonce;
   }
 
+  /**
+   * getFeeConfig from Godwoken RPC
+   */
+  public async getFeeConfig(): Promise<FeeConfig> {
```

Comment:
Do not use `bigint` in RPC interface directly, please use `HexNumber` instead.

Reply:
use HexU64 instead